student,q11,q11-failed,q31,q31-failed,q211,q211-failed,q21,q21-failed,total_grade,comments
test0_3,0.0,"['Exception raised:', '    Traceback (most recent call last):', '      File ""/anaconda3/lib/python3.7/doctest.py"", line 1329, in __run', '        compileflags, 1), test.globs)']",0.0,"['Exception raised:', '    Traceback (most recent call last):', '      File ""/anaconda3/lib/python3.7/doctest.py"", line 1329, in __run', '        compileflags, 1), test.globs)']",0.0,"['Expecting:', ""    \\'repeekkoob\\'"", '**********************************************************************', 'Line 8, in /Users/guoshumao/Downloads/19 Summer/Summer_2019/notebooks/tests/q211.py 0']",0.0,"['Failed example:', '    woman_asking', 'Exception raised:', '    Traceback (most recent call last):']",0.0,
test0_3,0.0,"['Exception raised:', '    Traceback (most recent call last):', '      File ""/anaconda3/lib/python3.7/doctest.py"", line 1329, in __run', '        compileflags, 1), test.globs)']",0.0,"['Exception raised:', '    Traceback (most recent call last):', '      File ""/anaconda3/lib/python3.7/doctest.py"", line 1329, in __run', '        compileflags, 1), test.globs)']",0.0,"['Expecting:', ""    \\'repeekkoob\\'"", '**********************************************************************', 'Line 8, in /Users/guoshumao/Downloads/19 Summer/Summer_2019/notebooks/tests/q211.py 0']",0.0,"['Failed example:', '    woman_asking', 'Exception raised:', '    Traceback (most recent call last):']",0.0,
test0_3,0.0,"['Exception raised:', '    Traceback (most recent call last):', '      File ""/anaconda3/lib/python3.7/doctest.py"", line 1329, in __run', '        compileflags, 1), test.globs)']",0.0,"['Exception raised:', '    Traceback (most recent call last):', '      File ""/anaconda3/lib/python3.7/doctest.py"", line 1329, in __run', '        compileflags, 1), test.globs)']",0.0,"['Expecting:', ""    \\'repeekkoob\\'"", '**********************************************************************', 'Line 8, in /Users/guoshumao/Downloads/19 Summer/Summer_2019/notebooks/tests/q211.py 0']",0.0,"['Failed example:', '    woman_asking', 'Exception raised:', '    Traceback (most recent call last):']",0.0,
test0_3,0.0,"['Exception raised:', '    Traceback (most recent call last):', '      File ""/anaconda3/lib/python3.7/doctest.py"", line 1329, in __run', '        compileflags, 1), test.globs)']",0.0,"['Exception raised:', '    Traceback (most recent call last):', '      File ""/anaconda3/lib/python3.7/doctest.py"", line 1329, in __run', '        compileflags, 1), test.globs)']",0.0,"['Failed example:', '    woman_asking', 'Exception raised:', '    Traceback (most recent call last):']",0.0,
test0_3,2.5,[],0.0,"['Exception raised:', '    Traceback (most recent call last):', '      File ""/anaconda3/lib/python3.7/doctest.py"", line 1329, in __run', '        compileflags, 1), test.globs)']",0.0,"['Expecting:', ""    \\'repeekkoob\\'"", '**********************************************************************', 'Line 8, in /Users/guoshumao/Downloads/19 Summer/Summer_2019/notebooks/tests/q211.py 0']",0.0,"['Failed example:', '    woman_asking', 'Expected:', ""    \\'The woman asked:\\'""]",2.5,
test0_3,2.5,[],0.0,"['Exception raised:', '    Traceback (most recent call last):', '      File ""/anaconda3/lib/python3.7/doctest.py"", line 1329, in __run', '        compileflags, 1), test.globs)']",0.0,"['Expecting:', ""    \\'repeekkoob\\'"", '**********************************************************************', 'Line 8, in /Users/guoshumao/Downloads/19 Summer/Summer_2019/notebooks/tests/q211.py 0']",0.0,"['Failed example:', '    woman_asking', 'Expected:', ""    \\'The woman asked:\\'""]",2.5,
test0_3,2.5,[],0.0,"['Exception raised:', '    Traceback (most recent call last):', '      File ""/anaconda3/lib/python3.7/doctest.py"", line 1329, in __run', '        compileflags, 1), test.globs)']",0.0,"['Expecting:', ""    \\'repeekkoob\\'"", '**********************************************************************', 'Line 8, in /Users/guoshumao/Downloads/19 Summer/Summer_2019/notebooks/tests/q211.py 0']",0.0,"['Failed example:', '    woman_asking', 'Expected:', ""    \\'The woman asked:\\'""]",2.5,
test0_3,2.5,[],0.0,"['Exception raised:', '    Traceback (most recent call last):', '      File ""/anaconda3/lib/python3.7/doctest.py"", line 1329, in __run', '        compileflags, 1), test.globs)']",0.0,"['Expecting:', ""    \\'repeekkoob\\'"", '**********************************************************************', 'Line 8, in /Users/guoshumao/Downloads/19 Summer/Summer_2019/notebooks/tests/q211.py 0']",0.0,"['Failed example:', '    woman_asking', 'Expected:', ""    \\'The woman asked:\\'""]",2.5,
